src/Microsoft.ML.Sweeper/AsyncSweeper.cs

```diff
@@ -47,7 +47,7 @@ public interface IAsyncSweeper
         /// Propose a <see cref="ParameterSet"/>.
         /// </summary>
         /// <returns>A future <see cref="ParameterSet"/> and its id. Null if unavailable or cancelled.</returns>
-        Task<ParameterSetWithId> Propose();
+        Task<ParameterSetWithId> ProposeAsync();
```

Comment:
📝 This is the only public API change in this pull request. It's not clear to me if/when this API shipped or whether it's allowed to be renamed at this point. If we can't rename it, I'll revert the change to this file and use a suppression instead.

Reply:
`Microsoft.ML.Sweeper` hasn't shipped in any nuget package, so I think we are safe to rename this.